src/templates/company/index.js

```diff
@@ -62,17 +66,20 @@ export default {
     ],
   ],
   prepareTransactions(createTx, data) {
+    const financePeriod = 0 // default
     const hasPayroll = false
+    const useAgentAsVault = false
 
-    const {
-      tokenName,
-      tokenSymbol,
-      subdomain,
-      holders,
-      stakes,
-      votingSettings,
-      financePeriod,
-    } = data
+    const { domain, tokens, voting } = data
```

Comment:
@bpierre I went ahead and grouped each configuration step into its own object and adjusted it in the three templates